packages/@aws-cdk/aws-elasticloadbalancingv2/README.md

```diff
@@ -77,6 +77,8 @@ const securityGroup2 = new ec2.SecurityGroup(this, 'SecurityGroup2', { vpc });
 lb.addSecurityGroup(securityGroup2);
 ```
 
+NOTE: Enabling ALB logs using an encryption enabled Amazon S3 Bucket is currently unsupported
```

Comment:
Are you sure this is the case? The [docs](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html#access-logging-bucket-requirements) state that

> With Network Load Balancer access logs, you can't use AWS managed keys, you must use customer managed keys.

Reply:
In the linked issue there's a reproduction repo provided that uses a bucket with encryption set to KMS - toggling this setting on and off will allow deployment to either fail or succeed. Unless I'm mistaken this setting is creating a CMK

Reply:
So, this is what the ALB doc says
```Create an S3 bucket
When you enable access logs, you must specify an S3 bucket for the access logs. The bucket must meet the following requirements.

Requirements

The bucket must be located in the same Region as the load balancer. The bucket and the load balancer can be owned by different accounts.

The prefix that you specify must not include AWSLogs. We add the portion of the file name starting with AWSLogs after the bucket name and prefix that you specify.

The only server-side encryption option that's supported is Amazon S3-managed keys (SSE-S3). For more information, see [Amazon S3-managed encryption keys (SSE-S3)](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html).

I can change the README to say - 
`NOTE: Enabling ALB logs using an encryption enabled Amazon S3 Bucket is only supported when using Amazon S3-managed keys (SSE-S3)`
Would that help?

Reply:
I believe the `BaseLoadBalancer` class is used by _both_ the `ApplicationLoadBalancer` and the `NetworkLoadBalancer`. If there is a difference between behavior then we need to handle it in the specific subclass.

Reply:
Yea, my bad, should have checked the inheritance tree. I can certainly move this change to the `ApplicationLoadBalancer` class and rejig the tests